### ISSUE TITLE
[BottomNavigation] a11y Color themer contrast

### ISSUE THREAD

**Issue by yarneo** [material-components/material-components-ios#4304]
## Expected behavior

Have enough contrast between the item colors and the navigation background color.

---
#### Internal data
- Associated internal bug: [b/117178862](http://b/117178862)

**Comment by romoore** [comment]
@yarneo  Could you please provide screenshots and reproduction steps for this bug?

**Comment by yarneo** [comment]
This wasn't posted by me in the sheet, I only provided a bug for it to track. Will be able to look into it, but not today.

**Comment by romoore** [comment]
@yarneo Gentle ping.

**Comment by jverkoey** [comment]
@yarneo Are you looking at this? Please unassign yourself if not.

**Comment by yarneo** [comment]
@romoore I think the intent here is that the unselected images and text of the bottom navigation don't have enough of a color contrast from the color of the navigation bar itself. as can be seen here:
<img width="395" alt="screen shot 2018-08-28 at 5 44 07 pm" src="https://user-images.githubusercontent.com/4066863/44752775-0bb0b780-aaea-11e8-8446-243a205b2bca.png">

Is this something we should take action on?

**Comment by romoore** [comment]
The colors I see rendered are:
* Background (#6200EE)
* Unselected Icon (#C099F8)

This gives a contrast ratio of 3.32:1 which is too low for small icons (one or both dimensions less than 20 points).

I've opened internal issue b/113495079 to request higher-contrast colors for the inactive items.

**Comment by jverkoey** [comment]
Un-assigning due to inactivity. Please re-assign if you are working on this.

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
